components/automate-ui/src/app/modules/team/team-details/team-details.component.spec.ts

```diff
@@ -65,21 +73,19 @@ describe('TeamDetailsComponent v2', () => {
   let router: Router;
   let store: Store<NgrxStateAtom>;
 
-  const v2PolicyEntityInitialState = Object.assign({}, PolicyEntityInitialState,
-    {iamMajorVersion: 'v2'});
   const initialState = {
     ...defaultInitialState,
     router: {
+      ...defaultRouterState,
       state: {
+        ...defaultRouterRouterState,
         url: `/settings/teams/${targetId}`,
-        params: { id: targetId },
-        queryParams: {},
-        fragment: '',
-        path: []
-      },
-      navigationId: 0 // what's that zero?
+        params: { id: targetId }
+      }
     },
-    policies: v2PolicyEntityInitialState
+    policies: {
+      ...PolicyEntityInitialState, iamMajorVersion: 'v2' as IAMMajorVersion
+    }
   };
```

Comment:
Removed more duplication for a few files in this commit, which also has the side benefit of showing just the fields that are relevant to the test.